src/main/java/net/mcreator/ui/views/editor/image/canvas/CanvasRenderer.java

```diff
@@ -204,6 +204,10 @@ public BufferedImage render() {
 	}
 
 	private void rebuildCheckerboardPattern() {
+		checkerboard = buildCheckerboardPattern();
+	}
+
+	public static TexturePaint buildCheckerboardPattern() {

```

Comment:
This may belong to some other class, maybe imageutils

Reply:
I left it here because it uses constants defined in this class. I don't think these constants belong in ImageUtils, but on the other hand leaving them here and making them public while their only reference is in ImageUtils wouldn't make much sense either.

Reply:
Ok fine then